package.json

```diff
@@ -22,7 +22,7 @@
     "test:coverage": "yarn run test:common && yarn run jest --coverage",
     "coverage": "yarn run test:coverage && open coverage/lcov-report/index.html",
     "styleguidist": "styleguidist server",
-    "start": "concurrently --kill-others-on-fail \"yarn run styleguidist\" \"webpack -w\"",
+    "start": "concurrently --kill-others-on-fail \"yarn run styleguidist\" \"webpack -w\" \"rollup -c rollup.config.js -w\"",
```

Comment:
`styleguidist` is using the `dist/es/index.js` files.  I decided to run both webpack and rollup in watch mode just in case.